lib/main.js

```diff
@@ -426,8 +426,8 @@ export default {
   consumeLinter(registry: any) {
     this.buildLinter = registry({ name: 'go build' })
     this.subscriptions.add(this.buildLinter)
-    this.gometalinterLinter = registry({ name: 'gometalinter' })
-    this.subscriptions.add(this.gometalinterLinter)
+    this.linterLinter = registry({ name: 'go linter' })
```

Comment:
s/linterLinter/linter/g

Reply:
`this.linter` already exists, which is the go-plus `Linter` instance. Maybe I should rename `linterLinter` to `linterProvider` or something else...